src/components/ForgotPasswordForm.vue

```diff
@@ -52,7 +52,7 @@ const appIcon = window.config.appIcon
                         <p>{{ $t('forgotPasswordForm.redirectToLoginMessage') }}</p>
                     </v-col>
                     <v-col cols="12" sm="5" class="text-sm-left">
-                        <router-link style="text-decoration: none; color: inherit;" class="font-weight-bold" to="/">{{
+                        <router-link style="text-decoration: none; color: inherit;" class="font-weight-bold" to="/system-admins-login">{{
```

Comment:
I think we should do /system-admins/login that's the pattern for other routes as well